Zend/zend_compile.c

```diff
@@ -1073,30 +1073,6 @@ ZEND_API int do_bind_function(zval *lcname) /* {{{ */
 }
 /* }}} */
 
-ZEND_API int do_bind_class(zval *lcname) /* {{{ */
```

Comment:
I'd prefer to normalize these under the name `do_bind_class`.

Reply:
Okay, rebased and adjusted. Since this is marked `ZEND_API` maybe we should rename it `zend_bind_class` though?